test/TestExpectations.json

```diff
@@ -2236,6 +2238,13 @@
     "parameters": ["cdp", "firefox"],
     "expectations": ["SKIP"]
   },
+  {
+    "testIdPattern": "[frame.spec] Frame specs Frame Management should send \"framenavigated\" when navigating on anchor URLs",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["firefox", "webDriverBiDi"],
+    "expectations": ["SKIP"],
+    "comment": "browsingContext.navigationStarted event not emitted for fragment navigation"
```

Comment:
It looks like it is spec-compliant. Let's restore the previous behavior?

Reply:
Added a comment. We can resolve this in another PR by trying to emulate `navigationStarted`.